### ISSUE TITLE
Include more links in show and movies details

### ISSUE THREAD

**Issue by jonathanantoine** [trakt/trakt-api#237]
Hello,

Is there a way to retrieve all the links displayed on the website (instagram, twitter, etc.) ?
![image](https://user-images.githubusercontent.com/3800550/102266343-9bc3d680-3f18-11eb-889c-c859958192b4.png)

Thanks !

**Comment by rectifyer** [comment]
No, we don't return that info in the API currently. We get it from TMDB, so you could go straight to the source? I guess it would need to be a new method on Trakt, I don't think it's worth adding to the `full` response.
